### PR TITLE
feat(dataloaders): specify seed types explicitly

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/930/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

File: pyTigerGraph/gds/dataloaders.py [hunk 1]
```diff
@@ -1621,20 +1622,29 @@ def __init__(
             self._etypes = list(self._e_schema.keys())
         self._vtypes = sorted(self._vtypes)
         self._etypes = sorted(self._etypes)
+        if v_seed_types:
+            if isinstance(v_seed_types, list):
+                self._seed_types = v_seed_types
+            elif isinstance(v_seed_types, str):
+                self._seed_types = v_seed_types
```
Comment (billshitg) [inline pyTigerGraph/gds/dataloaders.py:1629]: Should this be different from line 1627?

File: pyTigerGraph/gds/dataloaders.py [hunk 2]
```diff
@@ -3923,19 +3941,28 @@ def __init__(
         self._vtypes = sorted(self._vtypes)
         self._etypes = sorted(self._etypes)
         # Resolve seeds
+        if v_seed_types:
+            if isinstance(v_seed_types, list):
+                self._seed_types = v_seed_types
+            elif isinstance(v_seed_types, str):
+                self._seed_types = v_seed_types
```
Comment (billshitg) [inline pyTigerGraph/gds/dataloaders.py:3948]: same here

**Review by billshitg — APPROVED** [review]

File: pyTigerGraph/gds/dataloaders.py (hunk 1 above)
Comment (parkererickson-tg) [reply pyTigerGraph/gds/dataloaders.py]: Whoops, should be wrapped in a list.

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/933/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

**Review by billshitg — APPROVED** [review]

**Pull request merged**